package.json

```diff
@@ -38,9 +38,9 @@
     "build": "run-p -l build:*",
     "build:extension": "npm run compile:extension -- --minify",
     "build:webviews": "cd ./src/webviews && npm run build",
-    "dev": "run-p dev:*",
-    "dev:extension": "npm run verify-deps && npm run compile:extension -- --sourcemap --watch",
+    "dev": "run-s dev:*",
     "dev:webviews": "cd ./src/webviews && npm run dev",
+    "dev:extension": "npm run compile:extension -- --sourcemap --watch",
```

Comment:
@maninak don't forget to re-add `npm run verify-deps &&` prefix or sort out how to go on without it